### ISSUE TITLE
Broker returns text/plain type when Accept is */*

### ISSUE THREAD

**Issue by kiskit** [ScorpioBroker/ScorpioBroker#312]
Hello,

When sending a request ending in error, with an `Accept` header set to `*/*`, the response is sent with `Content-Type` set to `text/plain`. Shouldn't it be `application/json`? 

Example:
``` bash
curl --request GET --url http://127.0.0.1:9090/ngsi-ld/v1/entities/notavaliduri
```
Provides:
```json
{
  "type": "https://uri.etsi.org/ngsi-ld/errors/BadRequestData",
  "title": "Bad Request Data.",
  "details": "id is not a URI"
}
```
With Content-Type set to 
`text/plain;charset=UTF-8`

**Comment by martin-p-bauer** [comment]
You are right - the mime type shall be application/json according to the specification:

### 5.5.3	Error response payload body
When reporting errors back to clients, NGSI-LD implementations shall generate a JSON object in accordance with IETF RFC 7807 [10], section 3.1, including, at least the following terms:

- 	type: Error type as per clause 5.5.2.
- 	title: Error title which shall be a short string summarizing the error.
- 	detail: A detailed message that should convey enough information about the error.

Even though IETF RFC 7807 [10] defines a specific MIME type for error payloads, NGSI-LD implementations shall use the standard JSON MIME type ("**application/json**") when reporting errors, so that old clients or existing tools are not broken.

@developing team: please fix!

**Comment by amitrnec** [comment]
Hi,
Currently working on this issue.

Thanks.

**Comment by amitrnec** [comment]
Hi,
We have fixed this issue #313 

Thanks.

**Comment by amitrnec** [comment]
Hi,
Now we have merged in Dev branch.

Thanks.
